pkg/operatorsource/configuring.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 
 	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	v1 "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
```

Comment:
Why are we importing and aliasing  `"github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"` twice?